packages/compass-components/src/components/item-action-controls.tsx

```diff
@@ -31,21 +26,21 @@ const actionIconButton = css({
   },
 });
 
-export const ActionControls: React.FunctionComponent<{
-  actions: ActionControlOptions;
-  onAction(actionName: Actions): void;
-  className?: string;
-  isActive: boolean;
-  isHovered: boolean;
-  shouldCollapseActionsToMenu?: boolean;
-}> = ({
+export function ItemActionControls<Actions extends string>({
```

Comment:
@gribnoysup I couldn't figure out how to add the Actions type to make a generic function with the previously used syntax. Is this a problem? Any points on how to do this better?